File: Day_1/question1.c
#include<stdio.h>
int main(){
    int n, sum=0, i=0;

    printf("Enter the number you want till you wanted to find the sum: ");

    scanf("%d",&n);

    while(i<=n){

        sum = sum+i;

        i++;
    }
    
    printf("the sum of N natural numbers is: %d", sum);
    return 0;
}

File: Day_1/question1.py
#WAP to calculate sum of first N natural numbers.
n = int(input("Enter a positive integer: "))

if n <= 0:
    print("Please enter a number greater than 0.")
else:
    total_sum = 0
    for i in range(1, n + 1):
        total_sum += i

    print(f"The sum of the first {n} natural numbers is: {total_sum}")